lib/jxl/test_utils.h

```diff
@@ -273,25 +273,60 @@ std::vector<ColorEncodingDescriptor> AllEncodings() {
 std::vector<uint8_t> GetSomeTestImage(size_t xsize, size_t ysize,
                                       size_t num_channels, uint16_t seed) {
   // Cause more significant image difference for successive seeds.
-  std::mt19937 rng(seed);
-  std::uniform_int_distribution<uint16_t> dark(0, 32767);
+  std::mt19937 std_rng(seed);
+  std::uniform_int_distribution<uint16_t> std_distr(0, 65535);
+
+  // Returns random integer in interval (0, max_value - 1)
```

Comment:
Nit: uniform_int_distribution takes a closed interval, so this is in [0, max_value).

Reply:
Thanks for spotting that, dividing through 65536.0f instead should have fixed this

Reply:
Sounds good, thanks for making the change!